### PR TITLE
added docker-compose example

### LINKED ISSUE #56: docker-compose for Ubuntu
Is there any docker-compose for this STF?

### PR DESCRIPTION
**Pull request by lanrat**
Added example standalone docker-compose implementation.

Fixes #56.

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    container_name: rethinkdb
+    image: rethinkdb:2.3
+    restart: unless-stopped
+    volumes:
+        - "rethinkdb-data:/data"
+    command: "rethinkdb --bind all --cache-size 2048"
+
+  adb:
+    container_name: adb
+    image: sorccu/adb:latest
```
Comment (koral--) [inline docker-compose.yaml:14]:
```suggestion
    image: devicefarmer/adb:latest
```

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    volumes: 
+      - "/dev/bus/usb:/dev/bus/usb"
+    privileged: true
+
+  stf:
+    container_name: stf
+    image: devicefarmer/stf
+    ports:
+      - "7100:7100"
+        #- "7105:7105"
```
Comment (koral--) [inline docker-compose.yaml:25]: This comment adds no value.

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      - "7110:7110"
+        #- "7120:7120"
+      - "7400-7500:7400-7500"
+    environment:
+      - TZ='America/Los_Angeles'
+      - RETHINKDB_PORT_28015_TCP=tcp://rethinkdb:28015
+      - STF_ADMIN_EMAIL=YOUR_EMAIL
```
Comment (koral--) [inline docker-compose.yaml:32]: I would use something more eye-catching like <YOUR EMAIL>

File: docker-compose.yaml (hunk 2 above)
Comment (lanrat) [reply docker-compose.yaml]: comments removed.

File: docker-compose.yaml (hunk 3 above)
Comment (lanrat) [reply docker-compose.yaml:30]: Looks like your comment may have been cut off. Did you have something in particular to use as the placeholders?

Comment (koral--) [reply docker-compose.yaml:30]: yeah, something went wrong when posting the comment.
I mean `<YOUR EMAIL>`.

**Comment by koral--** [comment]
LGTM but please provide a sign off. DCO check won't pass otherwise.

File: docker-compose.yaml (hunk 3 above)
Comment (lanrat) [reply docker-compose.yaml]: Updated!

**Comment by lanrat** [comment]
As you can see by my plethora of recent commits/merges to my `compose` branch, I'm having some difficulty to get GitHub to accept the signed updated commit. I'll try a little more, but if I can't find the fix then I might just make a new CL.

**Pull request closed without merging**